src/social-network-provider/facebook.com/tasks/uploadToPostBox.ts

```diff
@@ -0,0 +1,32 @@
+import { SocialNetworkUI, getActivatedUI } from '../../../social-network/ui'
+import { untilDocumentReady } from '../../../utils/dom'
+import { getUrl, downloadUrl } from '../../../utils/utils'
+import Services from '../../../extension/service'
+
+export async function uploadToPostBoxFacebook(
+    this: SocialNetworkUI,
```

Comment:
suggestion: avoid the use of `this: SocialNetworkUI`, this pattern has encountered some problems in previous development